build/html/.buildinfo

```diff
@@ -0,0 +1,4 @@
+# Sphinx build info version 1
+# This file hashes the configuration used when building these files. When it is not found, a full rebuild will be done.
+config: c516599c870cca8cdc2de7ce72fdd1ba
+tags: 645f666f9bcd5a90fca523b33c5a78b7
```

Comment:
the build directory should not be committed.  remove and ignore

Reply:
done

Reply:
build directory is still there!!

Reply:
should be gone now with latest push (5/21/2020)